sdk/core/System.Net.ClientModel/api/System.Net.ClientModel.net6.0.cs

```diff
@@ -0,0 +1,407 @@
+        public static System.BinaryData Write<T>(T model, System.Net.ClientModel.ModelReaderWriterOptions? options = null) where T : System.Net.ClientModel.Core.IModel<T> { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ModelReaderWriterFormat : System.IEquatable<System.Net.ClientModel.ModelReaderWriterFormat>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ModelReaderWriterFormat(string value) { throw null; }
+        public static System.Net.ClientModel.ModelReaderWriterFormat Json { get { throw null; } }
+        public static System.Net.ClientModel.ModelReaderWriterFormat Wire { get { throw null; } }
+        public static System.Net.ClientModel.ModelReaderWriterFormat Xml { get { throw null; } }
+        public bool Equals(System.Net.ClientModel.ModelReaderWriterFormat other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object? obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(System.Net.ClientModel.ModelReaderWriterFormat left, System.Net.ClientModel.ModelReaderWriterFormat right) { throw null; }
+        public static implicit operator System.Net.ClientModel.ModelReaderWriterFormat (string value) { throw null; }
+        public static bool operator !=(System.Net.ClientModel.ModelReaderWriterFormat left, System.Net.ClientModel.ModelReaderWriterFormat right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class ModelReaderWriterOptions
+    {
+        public ModelReaderWriterOptions() { }
+        public ModelReaderWriterOptions(System.Net.ClientModel.ModelReaderWriterFormat format) { }
+        public static System.Net.ClientModel.ModelReaderWriterOptions DefaultWireOptions { get { throw null; } }
+        public System.Net.ClientModel.ModelReaderWriterFormat Format { get { throw null; } }
+    }
+    public partial class NullableResult<T> : System.Net.ClientModel.Result
+    {
+        internal NullableResult() { }
+        public virtual bool HasValue { get { throw null; } }
+        public virtual T? Value { get { throw null; } }
+        public override System.Net.ClientModel.Core.MessageResponse GetRawResponse() { throw null; }
+    }
+    public partial class RequestOptions : System.Net.ClientModel.Core.PipelineOptions
+    {
+        public RequestOptions() { }
+        public virtual System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
+        public static System.Threading.CancellationToken DefaultCancellationToken { get { throw null; } set { } }
```

Comment:
What's the difference between these two?

Reply:
The static one sets a global default, but the BCL team gave strong warnings about using statics like this, so we'll be removing this in the next PR into this branch.